src/main/java/frc/robot/commands/drivingCommands/DriveAndLoadingAimCommand.java

```diff
@@ -29,10 +31,11 @@ public void initialize() {
     @Override
     public void execute() {
         if (limelight.hasValidTarget()) {
+            slowSpeed = limelight.getTargetArea()/100;
```

Comment:
please make this 100.0 instead of 100 (also make it a constant, perhaps named TARGET_AREA_DIVISOR?

limelight.getTargetArea() is a double

100 is interpreted by Java as an integer.

It then has to convert that integer to a double before it can do the division.

In a program this size, this makes no difference in the execution time.  However, when you get to really big projects, each of these little conversions adds up and makes a difference.